source/config.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package source
+
+import (
+	"path/filepath"
+)
+
+var (
+	pathPrefix = "/"
```

Comment:
Should we set the default value to something that would fail to prevent reading the wrong directories if the variable is not set at build time?

Reply:
I was thinking about it and ended up with this to enable easier development experience. Building/running nfd-worker locally on a development system will cause it to do discovery on the standard system directories. Prefix is set in the Dockerfile to make container-based NFD behave differently (i.e. look under `/host-*`. Does this make sense?

Reply:
Oh ok, let's keep it like this then